Use non-canonical paths to populate the include paths in the toolchain in Bazel.
Otherwise, build will fail due to mismatch in include paths.
author: Alex Domingo (Vrije Universiteit Brussel)
--- third_party/gpus/cuda_configure.bzl.orig	2021-10-25 11:48:32.896463000 +0200
+++ third_party/gpus/cuda_configure.bzl	2021-10-25 11:51:56.719009257 +0200
@@ -274,8 +274,12 @@
     sysroot = []
     if tf_sysroot:
         sysroot += ["--sysroot", tf_sysroot]
-    result = raw_exec(repository_ctx, [cc, "-E", "-x" + lang, "-", "-v"] +
-                                      sysroot)
+
+    result = raw_exec(
+        repository_ctx,
+        [cc, "-no-canonical-prefixes", "-E", "-x" + lang, "-", "-v"] + sysroot
+    )
+
     stderr = err_out(result)
     index1 = stderr.find(_INC_DIR_MARKER_BEGIN)
     if index1 == -1:
